Sources/NIO/ByteCollectionUtils.swift

```diff
@@ -0,0 +1,42 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 2017-2018 Apple Inc. and the SwiftNIO project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftNIO project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+fileprivate let defaultWhitespaces = [" ", "\t"].map({UInt8($0.utf8CString[0])})
```

Comment:
`.utf8CString[0]` is not right, it should be `$0.utf8.first!`

Reply:
Ok, changed it